client/ayon_core/hosts/houdini/plugins/publish/extract_composite.py

```diff
@@ -45,8 +46,14 @@ def process(self, instance):
             "frameEnd": instance.data["frameEndHandle"],
         }
 
-        from pprint import pformat
-
-        self.log.info(pformat(representation))
+        if ext == "exr":
```

Comment:
```suggestion
        if ext.lower() == "exr":
```
I know we have it elsewhere and that it is total edge case, but ...